app/src/main/java/com/g11x/checklistapp/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.g11x.checklistapp;
+
+/**
+ * Created by wcalabrese on 8/15/16.
+ */
+
+public class Constants {
+    public static final String LOCAL_REPOSITORY_AUTHORITY = "com.g11x.checklistapp.provider";
```

Comment:
You can declare this in build.gradle as a res property and then use it in the manifest as well as within the app getString(context, R.string.thing).